src/classes/ALLO_Allocations_TEST.cls

```diff
@@ -531,7 +508,7 @@ private with sharing class ALLO_Allocations_TEST {
         setupSettings(new Allocations_Settings__c(Default_Allocations_Enabled__c = true, Default__c = defaultGau.id));
 
         //setting flags to not create default allocations for the new opportunities
-        ALLO_Allocations_TDTM.resetFlags(true);
+        TDTM_TriggerHandler.toggleTriggerState('Opportunity', 'ALLO_Allocations_TDTM', false);
```

Comment:
why turning off the trigger directly, rather than continuing to use flags/counters?